docs/installations/install-11.md

```diff
@@ -133,34 +133,5 @@ redirect_from: /books/how-to-and-guides/page/clean-installing-windows-11
 
 You have successfully installed Windows 11.
```

Comment:
Should probably remove the article before this, and but the "You have successfully installed Windows 11" Under the post -install guide